
### Name: ZHANG Feng (张丰)

Research field: bioinformatics, computational biology, biomedical data analysis

-----------

### Education & Work experience: 

* **Associate Professor & Program Lead**, Hainan Bielefeld University of Applied Sciences, China, 2025.03-present

* **Associate Professor**, Hainan Bielefeld University of Applied Sciences, China, 2025.02-2025.03

* **Research Associate Professor**, Shanghai Jiao Tong University School of Medicine, China, 2024.01-2025.01

* **Research Assistant Professor**, Shanghai Jiao Tong University School of Medicine, China, 2020.10-2023.12

* **Graduate Assistant & Visiting Scholar**, Cincinnati Children's Hospital Medical Center, USA, 2018.01-2020.01

* **PhD**, Biostatistics, School of Life Sciences, Fudan University, China, 2015.09-2020.06 

* **Summer School**, University of California, Los Angeles, USA, 2013.08-2013.09

* **BSc**, Information Security, School of Mathmatics, Shandong University, China, 2011.09-2015.06 

-----------

### Published over 30 research papers.

Selected Papers:

* YAP-TEAD regulates the super-enhancer network to control early surface ectoderm commitment, **Nucleic Acids Research**, 2026.01, [paper](https://doi.org/10.1093/nar/gkaf1285), co-corresponding author
  
* Single-nucleus profiling of mouse inner ear aging uncovers cell type heterogeneity and hair cell subtype-specific age-related signatures, **Cell Reports**, 2025.06, [paper](https://doi.org/10.1016/j.celrep.2025.115781), co-first author
  
* A Novel Prognostic Signature of Mitophagy-Related E3 Ubiquitin Ligases in Breast Cancer, **International Journal of Molecular Sciences**, 2025.02, [paper](https://doi.org/10.3390/ijms26041551), co-corresponding author

* Feature-weight based measurement of cancerous transcriptome using cohort-wide and sample-specific information, **Cellular Oncology**, 2023.10, {[code](https://github.com/jumphone/FWP), [paper](https://doi.org/10.1007/s13402-023-00879-6), [full-text](https://rdcu.be/dn9I1)}, corresponding author

* InferLoop: leveraging single-cell chromatin accessibility for the signal of chromatin loop, **Briefings in Bioinformatics**, 2023.05, {[code](https://github.com/jumphone/InferLoop), [paper](https://doi.org/10.1093/bib/bbad166)}, first & co-corresponding author
  
* FitDevo: accurate inference of single-cell developmental potential using sample-specific gene weight, **Briefings in Bioinformatics**, 2022.09, {[code](https://github.com/jumphone/FitDevo), [paper](https://doi.org/10.1093/bib/bbac293)}, first & co-corresponding author

* scMAGIC: accurately annotating single cells using two rounds of reference-based classification, **Nucleic Acids Research**, 2022.05, {[code](https://github.com/TianLab-Bioinfo/scMAGIC), [paper](https://doi.org/10.1093/nar/gkab1275)}, co-first author

* Unsupervised Inference of Developmental Directions for Single Cells Using VECTOR, **Cell Reports**, 2020.08, {[code](https://github.com/jumphone/Vector), [paper](https://doi.org/10.1016/j.celrep.2020.108069)}, first author

* A novel approach to remove the batch effect of single-cell data, **Cell Discovery**, 2019.09, {[code](https://github.com/jumphone/BEER), [paper](https://doi.org/10.1038/s41421-019-0114-x)}, first author

* PhenoPro: a novel toolkit for assisting in the diagnosis of Mendelian disease, **Bioinformatics**, 2019.02, {[code](https://github.com/jumphone/PhenoPro), [paper](https://doi.org/10.1093/bioinformatics/btz100)}, co-first author

* SPRINT: an SNP-free toolkit for identifying RNA editing sites, **Bioinformatics**, 2017.07, {[code](https://github.com/jumphone/SPRINT), [paper](https://doi.org/10.1093/bioinformatics/btx473)}, first author
    
-----------

### Sent 25 BiUH students to intern at various top universities and hospitals in Shanghai, Guangzhou, and Hangzhou.




